src/main/java/uk/gov/justice/hmpps/casenotes/controllers/CaseNoteController.java

```diff
@@ -182,6 +205,28 @@ public CaseNoteType updateCaseNoteSubType(
         return caseNoteService.updateCaseNoteSubType(parentType, subType, body);
     }
 
+    @DeleteMapping("/{offenderIdentifier}/{caseNoteId}")
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "Soft Deletes a case note", nickname = "soft delete case note")
+    @ApiResponses({
+            @ApiResponse(code = 404, message = "Offender or case note not found")})
+    public void softDeleteCaseNote(
+            @ApiParam(value = "Offender Identifier", required = true, example = "A1234AA") @PathVariable("offenderIdentifier") final String offenderIdentifier,
+            @ApiParam(value = "Case Note Id", required = true, example = "518b2200-6489-4c77-8514-10cf80ccd488") @PathVariable("caseNoteId") final UUID caseNoteId) {
+        caseNoteService.softDeleteCaseNote(offenderIdentifier, caseNoteId);
+    }
+
+    @DeleteMapping("/amendment/{offenderIdentifier}/{caseNoteAmendmentId}")
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "Soft Deletes a case note amendment", nickname = "soft delete case note amendment")
+    @ApiResponses({
+            @ApiResponse(code = 404, message = "Offender or case note not found")})
+    public void softDeleteCaseNoteAmendment(
+            @ApiParam(value = "Offender Identifier", required = true, example = "A1234AA") @PathVariable("offenderIdentifier") final String offenderIdentifier,
+            @ApiParam(value = "Case Note Amendment Id", required = true, example = "1") @PathVariable("caseNoteAmendmentId") final Long caseNoteAmendmentId) {
```

Comment:
is `1` an example of a case note amendment id?

Reply:
yes they are numbered sequentially starting from 1